### DISCUSSION TITLE
KMM Sample

### DISCUSSION THREAD

**Discussion by mochadwi** [sergio-sastre/AndroidUiTestingUtils#53] [Ideas]
Does this also has Compose Multiplatform sample? 🙏

**Comment by sergio-sastre** [comment]
Hi @mochadwi !
Unfortunately, I've not tried it with Compose Multiplatform.

Just to clarify, this library uses adb and ActivityScenarios, which are Android specific, therfore, it requires emulators/ devices to run tests with it.
But using this library for testing the Android part of your Compose Multiplatform app, should be possible.

Could you specify what are the issues you are having to use it with Compose Multiplatform?
Thanks!
